worker/worker_package.go

```diff
@@ -0,0 +1,92 @@
+package worker
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log/slog"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+
+	"github.com/opensvc/oc3/cache"
+	"github.com/opensvc/oc3/mariadb"
+)
+
+func (t *Worker) handlePackage(nodeID string) error {
+	ctx := context.Background()
+	ctx, _ = context.WithTimeout(ctx, time.Second*5)
+
+	_, err := t.Redis.HDel(ctx, cache.KeyPackagesPending, nodeID).Result()
```

Comment:
You can use here:
`err := t.Redis.HDel(ctx, cache.KeyPackagesPending, nodeID).Err()`